.eslintrc

```diff
@@ -9,10 +10,11 @@
   ],
   "env": {
     "browser": true,
-    "es6": true
+    "es6": true,
+    "jest/globals": true,
```

Comment:
This will think that jest globals are valid in all parts of the codebase - can we just configure this for test files? https://eslint.org/docs/user-guide/configuring#configuration-based-on-glob-patterns